frontend-react/src/hooks/UseCreateFetch.ts

```diff
@@ -40,7 +40,12 @@ function createTypeWrapperForAuthorizedFetch(
         // this system assumes that we want to be making authenticated
         // requests whenever possible
         if (!headers.authorization || headers.authorization.length < 8) {
-            console.warn("Unauthenticated request to ", options.url);
+            console.warn(
```

Comment:
fine to keep this change if it's still helpful to you, but maybe format the whole thing as a template string rather than mixing template string with argument based log calls. Or you know, pick one implementation and go with it, either way

Reply:
Oh, this is very much intentional.

I'm leveraging `console`'s ability to format objects but making each a separate parameter to the call verses trying to stringify it all in javascript.

The debugger also allows dev to dig arbitrarily deep into objects:

<img width="912" alt="image" src="https://user-images.githubusercontent.com/74203452/192879599-02787e40-4da9-4b20-9549-67183a25d961.png">

<img width="781" alt="image" src="https://user-images.githubusercontent.com/74203452/192879683-a36509b8-33d7-40df-93bb-57e9ae10437e.png">

Reply:
got it, cool.